en/docs/edit-the-code/code-actions.md

```diff
@@ -1 +1,192 @@
-# Code actions
\ No newline at end of file
+# Code actions
+
+The Ballerina VS Code extension provides code actions for the two scenarios below.
+
+1. Quick fixes for selected diagnostics
+2. Refactoring to improve code readability and suggest best practices
+
+### Quick fixes
+
+Quick fixes are suggested based on the diagnostics displayed at a given cursor position.
+The Ballerina VS Code extension provides a wide range of quick fixes to such diagnostics.
```

Comment:
Is this the exclusive list of quick fixes for diagnostics? If so, we can say be more specific and say something like the following:

```
The Ballerina VS Code extension provides quick fixes for [diagnostics](link to page) identified in your Ballerina code as explained below.
```

@praneesha

Reply:
We do not provide quick fixes for every diagnostic in ballerina. Instead, for a small percentage of diagnostics, quick fixes are provided.